### ISSUE TITLE
Insufficient contrast in form error state

### ISSUE THREAD

**Issue by MewenLeHo** [Orange-OpenSource/Orange-Boosted-Bootstrap#1918]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

(Coming from an a11y meeting with @Aniort)

The color contrast for `:invalid` `<input>`, `<select>`, and `<textarea>` elements is not sufficient between focused and not focused states.

### Reduced test cases

We need to find a better way to convey information.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

v5.3.0
